charts/ipfs/templates/ingress-api.yaml

```
@@ -14,6 +14,7 @@ metadata:
           {{ $key }}: {{ $value | quote }}
         {{- end }}
 spec:
+  ingressClassName: nginx
```

```suggestion
  ingressClassName: {{ .Values.ingress.className }} 
```
It would be great to move it to the values and not hardcore nginx